### DISCUSSION TITLE
hard disk مشکل تکمیل شدن

### DISCUSSION THREAD

**Discussion by masiihhh** [hiddify/Hiddify-Manager#884] [Q&A سوالات رایج]
با عرض خسته نباشید خدمت تیم هیدیفای عزیز

**Comment by masiihhh** [comment]
من hard disk سرورم کاملا پر شده و واسه آپدیت کردن پنل به مشکل خوردم. راه حلی پیشنهاد میکنید؟ البته به غیر از rebuild کردن سرور و نصب مجدد پنل

**Reply by drsat2006** [reply]
سرورتون رو خاموش کنین. از دیتاسنتری که سرور رو خریدید مقداری هارد خریداری کنین.و به هارد سیستمتون اضاف کنین. از سرور بکاپ کامل بگیرید . در صورتی که (با اضاف کردن هارد در دیتاسنتر ) حجم هارد شما افزایش پیدا نکرده بود.مجدد سرور رو نصب کنین و رستور بکاپ کنین.

**Comment by masiihhh** [comment]
![image](https://user-images.githubusercontent.com/103773770/235428191-46cfe5eb-ca37-4fc7-9a2d-bc7136f4ce23.png)
